Delicious/Scenes/Home/HomeViewController.swift

```diff
@@ -72,8 +75,37 @@ final class HomeViewController: UIViewController, BindableType {
     }
 
     func bindViewModel() {
-        let input = HomeViewModel.Input()
+        let error = tableView.rx.error { [weak self] in
+            self?.loadTrigger.onNext(())
+        }
+        let input = HomeViewModel.Input(
+            loadTrigger: loadTrigger.asDriverOnErrorJustComplete(),
+            reloadTrigger: tableView.loadMoreTopTrigger,
+            selectTrigger: selectTrigger.asDriverOnErrorJustComplete(),
+            searchTrigger: searchButton.rx.tap.asDriver()
+        )
+        
         let output = viewModel.transform(input)
+        
+        output.isLoading.drive(rx.isLoading).disposed(by: rx.disposeBag)
```

Comment:
```suggestion
        output.isLoading
            .drive(rx.isLoading)
            .disposed(by: rx.disposeBag)
```
Tương tự ở những chỗ khác nhé